osc-bsu-csi-driver/templates/controller.yaml

```diff
@@ -154,6 +175,27 @@ spec:
             - --v={{ .Values.verbosity }}
             - --leader-election=true
             - --timeout={{ .Values.timeout }}
+          {{- if .Values.sidecars.attacherImage.enableHttpEndpoint }}
+            - --http-endpoint=:{{ .Values.sidecars.attacherImage.httpEndpointPort }}
+          {{- end }}
+          {{- if .Values.sidecars.attacherImage.enableHttpEndpoint }}
+          ports:
+            - name: httpendpoint
+              containerPort: {{ .Values.sidecars.attacherImage.httpEndpointPort }}
+              protocol: TCP
+          {{- end }}
+          {{- if and .Values.sidecars.attacherImage.enableHttpEndpoint .Values.sidecars.attacherImage.enableLivenessProbe }}
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /healthz/leader-election
+              port: httpendpoint
+              scheme: HTTP
+            initialDelaySeconds: 10
```

Comment:
How do you choose you value ?

Reply:
Now, I use the default values. No need to configure more restrictive values

Reply:
Ok